pyproject.toml

```diff
@@ -36,6 +36,7 @@ dependencies = [
     "importlib_resources==6.4.0",
     "docstring_parser==0.16", # CLI help-formatter
     "rich_argparse==1.4.0", # CLI help-formatter
+    "setuptools<70",
```

Comment:
I think it's better to make an issue and add a comment to refer it for maintenance. How do you think about it?

Reply:
before filing an issue, need to find root cause of it. it's probably not related with otx itself so we need to clarify the actual target of the issue.

Reply:
There is relevant issue https://github.com/pypa/setuptools/issues/4376.
It seems that `setuptools` has a code `from pkg_resources.extern import packaging` in __init__.py before v70 but not include it in `__all__`.
So, it means that using `pkg_resources .packaging` isn't guaranteed and it's removed since v70.
Maybe it needs to be fixed by torch side, but I can't find an issue in torch repo.

Reply:
This seems to be an issue on the anomalib side. I'll open an issue.

Reply:
Add comments - https://github.com/openvinotoolkit/training_extensions/pull/3609/commits/f8462c6ce9db4d518d6069d97683992d54cf84bf